src/csgrep.cc

```diff
@@ -543,7 +574,8 @@ bool chainFilters(
         return false;
     }
 
-    return chainDecoratorIntArg<ImpLevelFilter>(pEng, vm, "imp-level");
+    return chainDecoratorIntArg<ImpLevelFilter>(pEng, vm, "imp-level")
+        && chainDecoratorGeneric<FingerPrintFilter>(pEng, vm, "hash-v1");
```

Comment:
nit: sort and align?
```suggestion
    return chainDecoratorGeneric<FingerPrintFilter>(pEng, vm, "hash-v1")
        && chainDecoratorIntArg <ImpLevelFilter>   (pEng, vm, "imp-level");
```

Reply:
Fixed in 83f4b2f18e15ae302d06d415097ed75ad2e41184.